_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -32,28 +30,33 @@ For simplicity the Request inside the Template only contains an AuthenticationRe
 Even though the Requests are validated during the RelationshipTemplate creation you should not skip this step as it gives you additional information in case of validation errors.
 {: .notice--info}
 
-## Create the Template
+## Create the RelationshipTemplate
 
-Create the Relationship Template on the Templator's Connector. You can do so by calling the `POST /api/v2/RelationshipTemplates/Own` route. Use the following JSON in the Request body:
+Next, [create a RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-own-relationshiptemplate.md %}) on the Sender Connector, using a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
+If there is no Relationship between the Connectors, yet, you must put the `content` property of the payload of the previous step in the `onNewRelationship` property of the RelationshipTemplateContent.
+However, if the Connectors already have a Relationship with each other, you must put it in the `onExistingRelationship` property of the `RelationshipTemplateContent`.
+But note, that the `onNewRelationship` property is required and must, therefore, always be set.
 
 ```jsonc
 {
   "maxNumberOfAllocations": 1,
-  "expiresAt": "2023-06-01T00:00:00.000Z",
+  "expiresAt": "2025-01-01T00:00:00.000Z",
   "content": {
     "@type": "RelationshipTemplateContent",
-    "title": "Connector Demo Contact",
+    "title": "Requests via RelationshipTemplates example",
     "onNewRelationship": {
-      // the content property of the payload in the step before
+      // the content property of the payload of the previous step
```

Comment:
I'd add a comma here after "step" and add the comment line "// if the Connectors do not already have a Relationship with each other" below to be consistent with the `onExistingRelationship` property.

I would also add a third line with "// ..." to be consistent with subsequent JSON payloads, here and for the `onExistingRelationship` property.

Reply:
We use "// ..." to indicate that some payload is left out in that place. Since this is not the case here, but in fact only the content property of the previous step shall be inserted and nothing in addition, I'd stick with these two lines.

Reply:
Maybe I thought more about the payloads here http://enmeshed.eu/integrate/establish-a-relationship-to-another-identity, where comments are also used just for comments and to indicate that something needs to be inserted. But in the latter case, also "// ..." is added there. Personally, I like this style. Do you have a general opinion on this?

Reply:
Ah I see. Honestly, I don't consider it to be that important. So I have no problem changing it.